pkg/server/jsonrpc.go

```diff
@@ -32,6 +32,9 @@ type spdkJSONRPC struct {
 // interact with either unix domain socket, e.g.: /var/tmp/spdk.sock
 // or with tcp connection ip and port tuple, e.g.: 10.1.1.2:1234
 func NewSpdkJSONRPC(socketPath string) JSONRPC {
+	if len(socketPath) == 0 {
```

Comment:
it is string so probably just `if socketPath == ""` is concise. The original thought in my head when I see `len` is that we work with arrays and in this case I need to go to func signature to understand that it is just string
strings in go cannot be `nil` so it is safe to compare against `""`

Reply:
ok